src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs

```diff
@@ -895,7 +895,7 @@ private static unsafe nuint GetIndexOfFirstNonAsciiChar_Vector(char* pBuffer, nu
             // Only run the loop if we have at least two vectors we can pull out.
             if (Vector512.IsHardwareAccelerated && bufferLength >= 2 * (uint)Vector512<ushort>.Count)
             {
-                const uint SizeOfVector512InChars = Vector512.Size / sizeof(ushort);
+                const uint SizeOfVector512InChars = Vector512.Size / sizeof(char);
```

Comment:
I don't think these changes are justified on readability grounds.

Reply:
Reverted the changes on this part.